wolfssl/internal.h

```diff
@@ -3002,6 +3002,9 @@ struct WOLFSSL_CTX {
     Srp*  srp;  /* TLS Secure Remote Password Protocol*/
     byte* srp_password;
 #endif
+#if defined(OPENSSL_EXTRA) && defined(HAVE_SECRET_CALLBACK)
+    wolfSSL_CTX_keylog_cb_func Keylog_cb;
```

Comment:
See note above, let's change "Keylog_cb" name to "keyLogCb" to match callback naming scheme in WOLFSSL_CTX.

Reply:
Changed to "keyLogCb".